kevinene91/Store-Manager-API
# #161216214 setup requirements and folder structure

### What does this PR do?
It has the Application folder structure, requirements, configurations, and the apps entry point 

### Description of Task?
Setup the app folder structure and its requirements 

### How to Test this Manually? 
- Clone the repository `git clone https://github.com/kevinene91/Store-Manager-API.git`
- Change into the app directory `cd Store-Manager-API`
- get this branch code `git fetch origin ch-setup-161216214`
- Navigate the folder structure
- create a virtual environment `python3 -m pyvenv venv`
- pip install -r requirements.txt
- python run.py development server should run

### Relevant PVT stories 
[161216214](https://www.pivotaltracker.com/n/projects/2203367/stories/161216214)
